documentation/player-portal/web/getting-started-web.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: page
+title: Getting Started with the Web Player
+weight: 110
+---
+
+The Kaltura HTML5 Player is really awesome and cool and fast and has nice features. 
+
+## Before You Begin 
+
+To create a new Kaltura Player, you'll need to have an active Kaltura Management Console (KMC) account. Click [here](https://corp.kaltura.com/Products/Video-Applications/Kaltura-Video-Management-Console) for more information about getting access.
+
+## Creating a TV Studio Player
+
+Kaltura Video Players are created and managed in the KMC [Studio](https://kmc.kaltura.com/index.php/kmcng/studio/v3), which is the second tab in the header. You'll find options for the Universal Studio, for legacy support, and the TV Platform Studio - which is colloquially referred to as Player V3. There you will also find all your available players in both categories. This guide will walk you through creating and embedding a TV Studio Player, which is focused on speed and performance.
```

Comment:
Studio V3 is not, at present time, enabled by default. It should be stated that if one cannot see it in KMC, one should contact us so that we may enable it.